runtime/oti/j9nonbuilder.h

```diff
@@ -262,6 +262,12 @@ typedef enum {
 	J9FlushCompQueueDataBreakpoint
 } J9JITFlushCompilationQueueReason;
 
+typedef enum {
+   BootstrapOnly               = 0,
+   BoostrapAndExtension        = 1,
+   UserDefined                 = 2
+} SharedClassCacheMode;
```

Comment:
* don't use anonymous types
* there's no need to explicitly assign values to the literals
* all these names live in the global namespace and need to be more specific
* this should include a value indicating the SCC is disabled (by default)

We want something more like this:
```
typedef enum J9SharedClassCacheMode {
    J9SharedClassCacheDisabled,
    J9SharedClassCacheBootstrapOnly,
    J9SharedClassCacheBoostrapAndExtension,
    J9SharedClassCacheUserDefined
} J9SharedClassCacheMode;
```

Reply:
When reaching the new API `j9shr_getSharedClassCacheMode`, shared cache cannot be inactive. We don't need to add `J9SharedClassCacheDisabled`

Reply:
Thanks for that information. My suggestion was based on the suggestion from @dsouzai which included the literal `INACTIVE`.

Reply:
Okay I'll keep Disabled option out, what do we prefer for the UserDefiined option?

Reply:
It should be as I showed above, without `J9SharedClassCacheDisabled`.